runtime/src/bank.rs

```diff
@@ -24,12 +24,15 @@ use solana_sdk::signature::{Keypair, Signature};
 use solana_sdk::system_transaction;
 use solana_sdk::timing::{duration_as_ms, duration_as_us, MAX_RECENT_BLOCKHASHES};
 use solana_sdk::transaction::{Result, Transaction, TransactionError};
+use solana_vote_api::vote_state::MAX_LOCKOUT_HISTORY;
 use solana_vote_api::vote_state::{self, Vote};
 use std::cmp;
 use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
 use std::sync::{Arc, RwLock};
 use std::time::Instant;
 
+pub const MINIMUM_SLOT_LEN: usize = MAX_LOCKOUT_HISTORY + 1;
```

Comment:
we have 2 conventions in the code base for names like this: _LEN  (in bench-tps and kvstore) and _LENGTH (as in repair_service).  I prefer LENGTH, but this is a nit.